MoEPluginInbox/MoEInboxBridge.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  MoEInboxBridge.swift
+//  MoEPluginInbox
+//
+//  Created by Rakshitha on 27/06/22.
+//
+
+import Foundation
+import MoEngageInbox
+import MoEPluginBase
+
+@objc public class MoEInboxBridge: NSObject {
```

Comment:
@RakshithaAcharya Is this a client facing class?

Reply:
@deepa-iOS-SDK  done